test/support/builders/metadata/base.js

```diff
@@ -158,22 +155,22 @@ module.exports = class BaseMetadataBuilder {
     return this
   }
 
-  updatedAt(date /*: Date */) /*: this */ {
-    this.doc.updated_at = timestamp.fromDate(date).toISOString()
+  updatedAt(date /*: string|Date */) /*: this */ {
+    this.doc.updated_at = typeof date === 'string' ? date : date.toISOString()
     return this
   }
 
   newerThan(doc /*: Metadata */) /*: this */ {
-    this.doc.updated_at = timestamp
-      .fromDate(new Date(timestamp.fromDate(doc.updated_at).getTime() + 2000))
-      .toISOString()
+    this.doc.updated_at = new Date(
+      new Date(doc.updated_at).getTime() + 2000
```

Comment:
I know it was there before but the 2000 looks like a magic number. I think it would be good to at least use a constant with a meaningful name for it.

Reply:
t is a magic number but we don't really care about its value. I'm not sure using a constant would convey the idea that it really doesn't matter what the value is.

I can try with something like SOME_TIME if you think that's better but I'm not really convinced.

Reply:
Could it describe why it's there (which i'm not super clear about either)? If I understand correctly it's an offset we use to force a document to be older or newer than another one?

Reply:
Yes that's it. It's just a helper function to create PouchDB records with a newer modification date than another record.